grails-app/services/org/pih/warehouse/inventory/RequisitionStatusTransitionEventService.groovy

```diff
@@ -33,4 +39,13 @@ class RequisitionStatusTransitionEventService {
             }
         }
     }
+
+    void onApplicationEvent(RequisitionStatusTransitionEvent event) {
+        requisitionService.triggerRequisitionStatusTransition(event.requisition, event.newStatus)
+
+    }
```

Comment:
Please, can you call `publishDefaultEmailNotifications` as it was mentioned in the tech huddle notes? I will be grateful.